### PR TITLE
feat(node/scripts/subscribe): allow use custom UA when updating subs

### PR DESCRIPTION
**Pull request by muink**
Used to resolve issue #188

### CODE REVIEW

File: htdocs/luci-static/resources/view/homeproxy/node.js [hunk 1]
```diff
@@ -1331,6 +1331,9 @@ return view.extend({
 		o.depends({'filter_nodes': 'disabled', '!reverse': true});
 		o.rmempty = false;
 
+		o = s.taboption('subscription', form.Value, 'custom_ua', _('Custom UA'));
```
Comment (1715173329) [inline htdocs/luci-static/resources/view/homeproxy/node.js:1334]:
```suggestion
		o = s.taboption('subscription', form.Value, 'user_agent', _('User-Agent'));
```

File: root/etc/homeproxy/scripts/homeproxy.uc [hunk 2]
```diff
@@ -74,11 +74,11 @@ export function getTime(epoch) {
 
 };
 
-export function wGET(url) {
+export function wGET(url, ua) {
 	if (!url || type(url) !== 'string')
 		return null;
 
-	const output = executeCommand(`/usr/bin/wget -qO- --user-agent 'Wget/1.21 (HomeProxy, like v2rayN)' --timeout=10 ${shellQuote(url)}`) || {};
+	const output = executeCommand(`/usr/bin/wget -qO- --user-agent ${shellQuote(ua || 'Wget/1.21 (HomeProxy, like v2rayN)')} --timeout=10 ${shellQuote(url)}`) || {};
```
Comment (1715173329) [inline root/etc/homeproxy/scripts/homeproxy.uc:81]:
```suggestion
	if (!ua)
		ua = 'Wget/1.21 (HomeProxy, like v2rayN)';

	const output = executeCommand(`/usr/bin/wget -qO- --user-agent ${shellQuote(ua)} --timeout=10 ${shellQuote(url)}`) || {};
```

File: root/etc/homeproxy/scripts/update_subscriptions.uc [hunk 3]
```diff
@@ -30,7 +30,8 @@ const ucimain = 'config',
       ucinode = 'node',
       ucisubscription = 'subscription';
 
-const allow_insecure = uci.get(uciconfig, ucisubscription, 'allow_insecure') || '0',
+const custom_ua = uci.get(uciconfig, ucisubscription, 'custom_ua'),
```
Comment (1715173329) [inline root/etc/homeproxy/scripts/update_subscriptions.uc:33]: please sort it alphabetically here

File: root/etc/homeproxy/scripts/update_subscriptions.uc [hunk 4]
```diff
@@ -31,6 +31,7 @@ const ucimain = 'config',
       ucisubscription = 'subscription';
 
 const allow_insecure = uci.get(uciconfig, ucisubscription, 'allow_insecure') || '0',
+      custom_ua = uci.get(uciconfig, ucisubscription, 'custom_ua'),
```
[the remark this answers is not in the document]
Comment (muink) [reply root/etc/homeproxy/scripts/update_subscriptions.uc]: !!OxO

**Pull request merged**